app/services/cads/live_data_service.rb

```diff
@@ -25,11 +27,13 @@ def power_for_type(powers, type)
     end
 
     def api
```

Comment:
We need some error handling in here to report API errors in Rollbar, even if they're not displayed to the user. I'd like to know if we've got authorisation or server errors so they can be caught when we're in production. This will help us debug issues with setup.

Reply:
If we get an auth error, then probably best to also invalidate the Rails cache to at least prompt another login attempt? Then if the token has expired then system will make some attempt to recover?

Reply:
And in the Rollbar logging can we add context so we know which school, device id, etc?